apiserver/vshn/postgres/delete.go

```diff
@@ -21,8 +21,7 @@ func (v vshnPostgresBackupStorage) Delete(_ context.Context, name string, _ rest
 	}, false, nil
 }
 
-func (v *vshnPostgresBackupStorage) DeleteCollection(_ context.Context, _ rest.ValidateObjectFunc, _ *metav1.DeleteOptions, _ *metainternalversion.ListOptions) (runtime.Object, error) {
-	return &v1.VSHNPostgresBackupList{
-		Items: []v1.VSHNPostgresBackup{},
-	}, nil
+func (v *vshnPostgresBackupStorage) DeleteCollection(ctx context.Context, _ rest.ValidateObjectFunc, _ *metav1.DeleteOptions, _ *metainternalversion.ListOptions) (runtime.Object, error) {
+	l, _ := v.List(ctx, &metainternalversion.ListOptions{})
```

Comment:
It's probably not a bad idea to do that but if you change that, please don't ignore the error and use the `ListOptions` provided

Reply:
we hage to rollback this